### PR TITLE
fix fishing-map errors ENG-444

### PR DESCRIPTION
**Pull request by fertandil87**

### CODE REVIEW

File: apps/port-labeler/package.json [hunk 1]
```diff
@@ -33,9 +33,9 @@
     "react-virtualized-auto-sizer": "^1.0.5",
     "react-window": "^1.8.6",
     "recoil": "0.7.5",
-    "redux": "4.2.0",
-    "redux-first-router": "2.1.5",
-    "redux-first-router-link": "2.1.1",
+    "redux": "^4.1.2",
```
Comment (rdgfuentes) [inline apps/port-labeler/package.json:36]: @fertandil87 Why are you using an older version of redux?

File: package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
   "workspaces": {
     "packages": [
       "apps/*",
-      "linting"
+      "linting",
+      "libs/i18n-labels"
```
Comment (rdgfuentes) [inline package.json:12]: @fertandil87 Why do we need to add this?

File: package.json [hunk 3]
```diff
@@ -32,6 +33,7 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@globalfishingwatch/i18n-labels": "*",
```
Comment (rdgfuentes) [inline package.json:36]: @fertandil87 Do we need this? why?

**Comment by j8seangel** [comment]
@fertandil87 I still had the same issue but updating all the dependencies has fixed it. Merging with the base branch to test everything there before merging

**Pull request merged**